src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.0.13",
+  "version": "3.0.14",
```

Comment:
remember to bump this again (or in the other PR, depending on which one you merge first 😄)